proto/expression.proto

```diff
@@ -269,7 +269,7 @@ enum ScalarFuncSig {
 	LEString = 113;
 	LETime = 114;
 	LEDuration = 115;
-	LEJson = 116;
+	LEJSON = 116;
```

Comment:
**CI of TiDB** requires `JSON` instead of `Json`, but **CI of TiKV** requires `Json` instead of `JSON`. For history reasons we use `Json` in tipb.